go/vt/schemamanager/tablet_executor.go

```diff
@@ -114,23 +116,26 @@ func (exec *TabletExecutor) Validate(ctx context.Context, sqls []string) error {
 	return err
 }
 
-func (exec *TabletExecutor) parseDDLs(sqls []string) ([]*sqlparser.DDL, error) {
+func (exec *TabletExecutor) parseDDLs(sqls []string) ([]*sqlparser.DDL, []*sqlparser.DBDDL, error) {
 	parsedDDLs := make([]*sqlparser.DDL, 0, len(sqls))
```

Comment:
Since we no longer know how many object parsedDDLs will contain, we shouldn't specify max size of the array.

Reply:
Done.